packages/tools/fluid-runner/src/codeLoaderBundle.ts

```diff
@@ -29,9 +29,10 @@ export interface IFluidFileConverter {
 	getCodeLoader(logger: ITelemetryBaseLogger): Promise<ICodeDetailsLoader>;
 
 	/**
-	 * Scope object to provide at Loader creation
+	 * Get scope object to provide at Loader creation
+	 * @param logger - created logger object to pass to scope object
 	 */
-	scope?: FluidObject;
+	getScope?(logger: ITelemetryBaseLogger): Promise<FluidObject>;
```

Comment:
That's technically a breaking change. And our tools do not pick it up as such.
Worth following up, as we need to get better long term not to break our clients in minor releases.
But now it's probably fine to be in main, assuming that Loop does not use it today.